lib/api/controllers/security.js

```diff
@@ -1505,11 +1505,15 @@ function mDelete (kuzzle, type, request) {
           errorsManager.get('services', 'storage', 'incomplete_delete', errors));
       }
 
+      const userId = request.context && request.context.user && request.context.user._id
```

Comment:
Why don't use `getUserId` here?

Reply:
Because this function is outside the class

Reply:
The security controller inherits from the NativeController which inherits from the BaseController which contain this method

Reply:
Oh yes you're right!
This function should be a method of the controller instead so you won't need to pass the kuzzle object to it and you can use the `getUserId` method